schema_utils_test.py

```diff
@@ -163,6 +168,9 @@
         'is_regex': {},
         'is_valid_email': {},
     },
+    SCHEMA_TYPE_CUSTOM: {
```

Comment:
Ditto. Why would all custom schemas have such a validation? There are a lot of different custom schemas and we usually handle "custom" as a special case.

Reply:
It not not necessary for all custom schema to have such a validation as seen be seen [here](https://github.com/oppia/oppia/blob/develop/schema_utils_test.py#L295).

Reply:
If you're making a chance to just one custom schema then try to contain that change to affect only that schema, don't make it spill over into the more general logic.

"Custom" kinda means that the validation etc. is delegated to the stuff specified by that schema.

Reply:
Done.